src/common/util/feature-flags.js

```diff
@@ -43,6 +43,9 @@ export function activateFeatures (flags, agentIdentifier) {
     }
   })
   sentIds.add(agentIdentifier)
+
+  // let any window level subscribers know that the agent is running
+  dispatchGlobalEvent({ loaded: true, agentIdentifier })
```

Comment:
What about sending the agent instance instead of the identifier? Thinking about the fact that this will be like a public API and difficult to change later.

Reply:
will modify this to only emit the loaded: true portion until bigger architecture changes are made to enable the agent instance comment